pkgs/development/python-modules/pyobjc/default.nix

```diff
@@ -0,0 +1,173 @@
+{ stdenv, fetchurl, pkgs, buildPythonPackage }:
+
+let
+
+  buildPyObjcFramework = _name: version: sha256: extraDeps:
+    buildPythonPackage rec {
+      name = "${pname}-${version}";
+      pname = "pyobjc-framework-${_name}";
+
+      propagatedBuildInputs = [ pyobjc-core ]
+        ++ stdenv.lib.optional (builtins.hasAttr _name pkgs.darwin.apple_sdk.frameworks) (builtins.getAttr _name pkgs.darwin.apple_sdk.frameworks)
+        ++ extraDeps;
+
+      patchPhase = ''
+        # hack to force 10.9 sdk
+        substituteInPlace pyobjc_setup.py \
+          --replace '"-DPyObjC_BUILD_RELEASE=%02d%02d"' '"-DPyObjC_BUILD_RELEASE=1009"]#'
+      '' + stdenv.lib.optionalString (_name == "AVFoundation") ''
+        # messy patches (fixed in 3.2)
+        sed -i -e '11i#if PyObjC_BUILD_RELEASE >= 1010' -e '15i#endif' Modules/_AVFoundation_inlines.m
+        sed -i -e '20i#if PyObjC_BUILD_RELEASE >= 1010' -e '22i#endif' Modules/_AVFoundation_protocols.m
+      '';
+
+      doCheck = false;
+
+      src = fetchurl {
+        url = "mirror://pypi/p/${pname}/${name}.tar.gz";
+        inherit sha256;
+      };
+
+      meta = {
+        platforms = stdenv.lib.platforms.darwin;
+        description = "Wrappers for the framework ${_name} on Mac OS X";
+        maintainers = [ stdenv.lib.maintainers.matthewbauer ];
+        license = stdenv.lib.licenses.mit;
+      };
+    };
+
+  frameworks = rec { # 3.1.1 corresponds to SDK 10.9 because we only have CF 10.9
+    Accounts = buildPyObjcFramework "Accounts" "3.1.1" "1jgasyz88mhl58ks1k06qqxd6ifzbkcjkir6ny0hjr6vc07lk99h" [ Cocoa ];
```

Comment:
How do you deal with updates of these packages? Do you edit these hashes manually or automatically?